src/components/cards/TripCard/TripCard.styles.js

```diff
@@ -0,0 +1,14 @@
+export const root =
+  'su-group su-relative su-overflow-hidden su-bg-black-true su-break-words su-border-su-border-black';
+export const content = 'su-rs-px-1 su-rs-pb-3 su-text-white';
```

Comment:
```suggestion
export const content = 'su-rs-px-2 su-rs-pb-3 su-text-white';
```
For now, let's use a padding-2 so the content padding matches the event. On the event cards, the dateblock is actually pulled more to the left compared to the content, so I'm asking Bhavika to confirm this. This is her most updated mockup
![Screen Shot 2021-09-14 at 4 45 22 PM](https://user-images.githubusercontent.com/42749717/133349378-c8357985-621c-43fb-ad73-212f19d5cf77.png)

It's still different from events, but for now let's just do a su-rs-px-2 on everything.

Reply:
Also, if you could please make the month and day left aligned in the dateblock, that'd be great (this part we're sure about)